auth/providers/azure/pop_tokenverifier_test.go

```diff
@@ -219,4 +231,13 @@ func TestPopTokenVerifier_Verify(t *testing.T) {
 		_, err := verifier.ValidatePopToken(invalidToken)
 		assert.Containsf(t, err.Error(), "Token is expired", "Error message is not as expected")
 	})
+
+	t.Run("'nonce' claim been reused.", func(t *testing.T) {
```

Comment:
Add a test to verify that you can reuse a nonce claim once the cache entry expires.